spec/unit/system_description_spec.rb

```diff
@@ -555,26 +555,6 @@ class SystemDescriptionFooConfig < Machinery::Object; end
     end
   end
 
-  describe "#has_file?" do
-    let(:system_description) {
-      create_test_description(json: <<-EOF)
-        {
-          "config_files": {
-            "extracted": true,
-            "files": [
-              {
-                "name": "/etc/my.cnf"
-              }
-            ]
-          }
-        }
-      EOF
-    }
-    it "returns true when found" do
-      expect(system_description.has_file?("/etc/my.cnf")).to be_truthy
-    end
-  end
```

Comment:
@cornelius wouldn't the test in scope_file_access_archive_spec.rb be enough in this case?